pom.xml

```diff
@@ -241,19 +241,16 @@ SOFTWARE.
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
-        <version>2.22.2</version>
-        <dependencies>
-          <dependency>
-            <groupId>org.apache.maven.surefire</groupId>
-            <artifactId>surefire-junit-platform</artifactId>
-            <version>2.22.2</version>
-          </dependency>
-        </dependencies>
+        <version>3.0.0-M5</version>
       </plugin>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-failsafe-plugin</artifactId>
+        <version>3.0.0-M5</version>
```

Comment:
Same here - you can remove it, since it's defined in parent pom

Reply:
I noticed that integration tests doesn't run with the version configured in parent pom (version `3.0.0-M4`). We got this :
```cmd
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running com.artipie.asto.etcd.EtcdStorageITCase
[INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.685 s - in com.artipie.asto.etcd.EtcdStorageITCase
```
Tests run when we use version `3.0.0-M5`.

Reply:
I see, will fix it with parent pom update later